dvc/command/check_ignore.py

```diff
@@ -0,0 +1,71 @@
+import argparse
+import logging
+
+from dvc.command import completion
+from dvc.command.base import CmdBase, append_doc_link
+from dvc.exceptions import DvcException
+
+logger = logging.getLogger(__name__)
+
+
+class CmdCheckIgnore(CmdBase):
+    def __init__(self, args):
+        super().__init__(args)
+        self.ignore_filter = self.repo.tree.dvcignore
+
+    def _show_results(self, results):
+        for result in results:
+            if result.matches or self.args.non_matching:
+                #
+                if self.args.details:
+                    logger.info(
+                        "{}\t{}".format(result.patterns[-1], result.file)
+                    )
+                else:
+                    logger.info(result.file)
+
+    def run(self):
+        if self.args.non_matching and not self.args.details:
+            raise DvcException("--non-matching is only valid with --details")
+
+        if self.args.quiet and self.args.details:
+            raise DvcException("cannot both --details and --quiet")
+
+        results = self.ignore_filter.check_ignore(self.args.targets)
+        self._show_results(results)
+        if any(result.matches for result in results):
+            return 0
+        return 1
+
+
+def add_parser(subparsers, parent_parser):
+    ADD_HELP = "Debug DVC ignore/exclude files"
+
+    parser = subparsers.add_parser(
+        "check-ignore",
+        parents=[parent_parser],
+        description=append_doc_link(ADD_HELP, "check-ignore"),
+        help=ADD_HELP,
+        formatter_class=argparse.RawDescriptionHelpFormatter,
+    )
+    parser.add_argument(
+        "-d",
+        "--details",
+        action="store_true",
+        default=False,
+        help="Show the exclude pattern together with the path.",
```

Comment:
```suggestion
        help="Show the exclude pattern together with each target path.",
```
Per iterative/dvc.org/pull/1629

Reply:
From iterative/dvc.org/pull/1629:

> All fields in each line, except for `<target path>`, will be empty.

So why even use the special --details format? Maybe just output the target paths normally with this option? (Does it even require --details?)